web-demo-app/src/app/conversation/conversation.module.ts

```diff
@@ -258,7 +258,7 @@ export class ConversationModule {
       'Send',
       'Type your message here...',
       '',
-      () => this.room.indicateTyping()
+      (value) => this.room.indicateTyping(value)
```

Comment:
We don't have to use brackets if there is single arrow function argument without an explicit type.